model/src/w3profsmd_pdlib.F90

```diff
@@ -127,6 +127,10 @@ MODULE PDLIB_W3PROFSMD
   LOGICAL, SAVE         :: LINIT_OUTPUT = .TRUE.
   REAL, SAVE            :: RTIME = 0.d0
   INTEGER               :: POS_TRICK(3,2)
+  REAL*8, PARAMETER     :: KDMAX     = 300.d0
+  REAL*8, PARAMETER     :: PI = 3.14159265358979323846D0
```

Comment:
This is defined in constants.   Is it possible to use those values instead of defining this in multiple places?